R/survival.R

```diff
@@ -59,30 +57,31 @@
 #' @examples
 #' survival_est <- bbr_survival(
 #'   bboudata::bbousurv_a,
-#'   mort_type = "total",
+#'   include_uncertain_morts = TRUE,
 #'   variance = "greenwood"
 #' )
 #' survival_est <- bbr_survival(
 #'   bboudata::bbousurv_b,
-#'   mort_type = "certain",
+#'   include_uncertain_morts = FALSE,
 #'   variance = "cox_oakes"
 #' )
-bbr_survival <- function(x, mort_type = "total", variance = "greenwood", year_start = 4L) {
+
+bbr_survival <- function(x, include_uncertain_morts = FALSE, variance = "greenwood", year_start = 4L) {
```

Comment:
default include_uncertain_morts should be TRUE (to match previous and bboutools functionality)

Reply:
Good catch thank you, changes pushed.